ui/src/components/Setting.tsx

```diff
@@ -34,7 +37,12 @@ export default function Setting({
           loading={status === 'loading'}
         />
         <div className="flex flex-1 flex-col justify-center">
-          <h3 id={id} className="flex items-center font-semibold leading-6">
+          <h3
+            id={id}
+            className={cn('flex items-center leading-6', {
+              'font-semibold': bold,
```

Comment:
maybe we should just open className above this and remove the font-semibold altogether and add it to anywhere setting was used?

Reply:
good idea, just made that change, will merge now